localization/en/strings.json

```diff
@@ -68,7 +68,9 @@
     "loading": "Loading...",
     "selling": "Selling...",
     "quick_sell": "Quick Sell for __amount__",
+    "quick_sell_desc": "Price difference of __modifier__ than the lowest selling offer price currently listed on the Steam Market",
```

Comment:
```suggestion
    "quick_sell_desc": "Price difference of __modifier__ compared to the lowest priced offer currently listed on the Steam Market",
```

Reply:
the subject is the price, not the offer.

Reply:
I don't think that there is such a big difference between a price and an offer here.
I did this change because your sentence is too confusing

Reply:
Okay, fair point. I'll improve it.